package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bufferapp/components",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
Should this be 2.1.1 or 2.2.0? 🤔

Reply:
Hey there @msanroman ... because we're not really using Semantic Versioning (because 2.X.X is a constant here) and because this didn't really include any breaking changes, I presumed that a X.X.1 would be enough?